dll/win32/shell32/dialogs/filedefext.cpp

```diff
@@ -1353,6 +1377,15 @@ CFileDefExt::CountFolderAndFiles(HWND hwndDlg, LPWSTR pwszBuf, UINT cchBufMax, D
             FileSize.u.LowPart  = wfd.nFileSizeLow;
             FileSize.u.HighPart = wfd.nFileSizeHigh;
             m_DirSize.QuadPart += FileSize.QuadPart;
+            // Calculate size on disc
+            if( GetVolumePathName(pwszBuf, lpszVolumePathName, sizeof(lpszVolumePathName) / sizeof(TCHAR)) ) {
+                if( GetDiskFreeSpace(lpszVolumePathName, &lpSectorsPerCluster, &lpBytesPerSector, NULL, NULL) )
```

Comment:
`GetDiskFreeSpaceEx` is better.

Reply:
No, the right solution is to use DeviceIoControl with IOCTL_DISK_GET_DRIVE_GEOMETRY. I add this to the TODO list